Mustack/node-jet
# [KAA-971] Add peerId concept to peers

### Story

[KAA-971](http://jira.genband.com/browse/KAA-971): Add peerId concept to peers in node-jet.

### Description

- The `Daemon` generates a `peerId` value that wasn't being passed along to the `Peer` on the client side
- This solves that by fetching the id before the `Peer` connects

### Tests

- Symlink this version of `node-jet` to use within `workout`
- Within `packages/workout-js-lib/src/workout.js`, add a `console.log(peer)`
- Run `yarn dev` within the `workout` directory
- Open http://localhost:8090/ and check the console
- You should see the `Peer` object with an `id` property
![image](https://user-images.githubusercontent.com/15205001/42053390-5d123c2e-7adf-11e8-9ab5-52e2bcdcc320.png)
